boards/arm/am62x_m4/doc/am62x_m4_phyboard_lyra.rst

```diff
@@ -0,0 +1,148 @@
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| PINCTRL   | on-chip    | pinctrl                             |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial                              |
++-----------+------------+-------------------------------------+
+
+Other hardware features are not currently supported by the port.
+
+Devices
+========
+System Clock
+------------
+
+This board configuration uses a system clock frequency of 400 MHz.
+
+DDR RAM
+-------
+
+The board has 2GB of DDR RAM available. This board configuration
+allocates Zephyr 4kB of RAM (only for resource table: 0x9CC00000 to 0x9CC00400).
+
+Serial Port
+-----------
+
+This board configuration uses a single serial communication channel with the
+MCU domain UART (MCU_UART0).
+
+SD Card
+*******
+
+Download PHYTEC's official `WIC`_ as well as `BMAP`_ and flash the WIC file with
+an etching software onto an SD-card. This will boot Linux on the A53 application
+cores of the SoM. These cores will then load the zephyr binary on the M4 core
+using remoteproc.
+
+The default configuration can be found in the defconfig file:
+
+.. code-block:: console
+
+   boards/arm/am62x_m4/am62x_m4_phyboard_lyra_defconfig
+
+Flashing
+********
+
+The Linux running on the A53 uses the remoteproc framework to manage the M4F co-processor.
+Therefore, the testing requires the binary to be copied to the SD card to allow the A53 cores to
+load it while booting using remoteproc.
+
+To test the M4F core, we build the `hello_world` sample with the following command.
+
+.. code-block:: console
+
+   # From the root of the Zephyr repository
+   west build -p -b am62x_m4_phyboard_lyra samples/hello_world
+
+This builds the program and the binary is present in the `build/zephyr` directory as `zephyr.elf`.
+
+We now copy this binary onto the SD card in the `/lib/firmware` directory and name it as `am62-mcu-m4f0_0-fw`.
+
+.. code-block:: console
+
+   # Mount the SD card at sdcard for example
+   sudo mount /dev/sdX sdcard
+   # copy the elf to the /lib/firmware directory
+   sudo cp --remove-destination zephyr.elf sdcard/lib/firmware/am62-mcu-m4f0_0-fw
+
+The SD card can now be used for booting. The binary will now be loaded onto the M4F core on boot.
+
+To allow the board to boot using the SD card, set the boot pins to the SD Card boot mode. Refer to `phyBOARD SD Card Booting Essentials`_.
+
+After changing the boot mode, stop in U-Boot to enable the M4F co-processor.
+
+.. code-block:: console
+
+   setenv overlays k3-am62-phyboard-lyra-rpmsg.dtbo
```

Comment:
interesting, what does the overlay do? Why is it required? Perhaps you can add some link to the source of this overlay in Documentation?

Reply:
I am assuming that you probably "disable" MCU core in your default DT? Any reason to do that by default? Maybe we can just enable it from default DT instead of needing an overlay?